CMakeLists.txt

```diff
@@ -190,6 +190,10 @@ if(NOT WIN32)
     $<$<COMPILE_LANGUAGE:CXX>:-Woverloaded-virtual>
     $<$<COMPILE_LANGUAGE:CXX>:-fno-operator-names>
   )
+else()
+  add_compile_options(
+    $<$<COMPILE_LANGUAGE:CXX>:/W4> $<$<COMPILE_LANGUAGE:CXX>:/bigobj>
```

Comment:
What does `/bigobj` do? And why did we not have it before?

Reply:
Bigobj increases the number of objects you can store in a `.obj` file (https://docs.microsoft.com/en-us/cpp/build/reference/bigobj-increase-number-of-sections-in-dot-obj-file?view=msvc-160). The build was warning about this previously.